rhizome/host/lib/vm_setup.rb

```diff
@@ -655,9 +671,20 @@ def install_systemd_unit(max_vcpus, cpu_topology, mem_gib, storage_params, nics,
 LimitNOFILE=500000
 #{limit_memlock}
 SERVICE
+    r "systemctl set-property #{q_vm_service} CPUQuota=#{cpu_percent_limit}%" unless cpu_percent_limit == 0
```

Comment:
instead of running this as a separate command, can we build the string and put it into the systemd service like we do for `#{limit_memlock}`